net/iodine/patches/101-musl-workaround-incomplete-nameser-h.patch

```diff
@@ -14,14 +14,20 @@ the `osflags` file.
 An upstream fix for musl libc has been submitted with
 http://www.openwall.com/lists/musl/2017/12/04/3
 
+The fix had since been merged, and 22 months later been removed again,
+noting that it should not be required anymore in the future:
+https://git.musl-libc.org/cgit/musl/commit/include/arpa/nameser.h?id=71d23fbeacdd1f0c5e33cffb0979250d576fcf9c
+
+So this patch can likely be removed. Testing is required, though.
```

Comment:
Then drop it and we will see.

Reply:
Dropped and incremented pkg_release version. Is that right? Or would the pkg_release need to be incrementend anyways, in case the version without that patch does not correctly work on big endian systems?

Reply:
It depends on the situation. You did it it alright.